From #263880:

The `shop' entry has an example phrase `go shop ping' at point 2.1, which I
believe should be `go shopping'.

Index: mueller/Mueller7accentGPL.koi
===================================================================
--- mueller.orig/Mueller7accentGPL.koi	2009-02-17 13:59:50.000000000 +0600
+++ mueller/Mueller7accentGPL.koi	2009-02-17 13:59:56.000000000 +0600
@@ -36860,7 +36860,7 @@
 shootinggallery  [хSuЫtINгgQl╚rI] _n. тир 
 shootingrange  [хSuЫtINreIndZ] _n. стрЕльбище, полигОн 
 shootingstick  [хSuЫtINstIk] _n. трость-сидЕнье 
-shop  [S█p] 1. _n. 1> лАвка, магазИн 2> мастерскАя, цех; closed shop _ам. предприЯтие, принимАющее на рабОту тОлько члЕнов профсоЮза; open shop _ам. предприЯтие, принимАющее на рабОту как члЕнов, так и нечлЕнов профсоЮза 3> _разг. заведЕние, учреждЕние, предприЯтие 4> профЕссия, занЯтие; делА, вопрОсы, тЕмы, свЯзанные с чьей-л. профЕссией; stop thinking of shop! хвАтит дУмать о делАх или рабОте!; to talk shop говорИть о делАх, говорИть на узкопрофессионАльные тЕмы во врЕмя Общего разговОра (в гостях и т.п.) 5> _attr. цеховОй; shop committee цеховОй комитЕт; shop chairman _ам. цеховОй стАроста; all over the shop разбрОсанный повсЮду, в беспорЯдке; to come to the wrong shop обратИться не по Адресу; to get a shop _театр. получИть ангажемЕнт; to lift a shop совершИть крАжу в магазИне; to shut up shop закрЫть лАвочку; прекратИть дЕятельность 2. _v. 1> дЕлать покУпки (обыкн. go shop ping) 2> _ам. ходИть по магазИнам, чтОбы ознакОмиться с цЕнами, присмотрЕть вещь 3> _жарг. сажАть в тюрьмУ 4> _жарг. выдавАть (сообщника); shop around _ам. искАть рабОту, мЕсто 
+shop  [S█p] 1. _n. 1> лАвка, магазИн 2> мастерскАя, цех; closed shop _ам. предприЯтие, принимАющее на рабОту тОлько члЕнов профсоЮза; open shop _ам. предприЯтие, принимАющее на рабОту как члЕнов, так и нечлЕнов профсоЮза 3> _разг. заведЕние, учреждЕние, предприЯтие 4> профЕссия, занЯтие; делА, вопрОсы, тЕмы, свЯзанные с чьей-л. профЕссией; stop thinking of shop! хвАтит дУмать о делАх или рабОте!; to talk shop говорИть о делАх, говорИть на узкопрофессионАльные тЕмы во врЕмя Общего разговОра (в гостях и т.п.) 5> _attr. цеховОй; shop committee цеховОй комитЕт; shop chairman _ам. цеховОй стАроста; all over the shop разбрОсанный повсЮду, в беспорЯдке; to come to the wrong shop обратИться не по Адресу; to get a shop _театр. получИть ангажемЕнт; to lift a shop совершИть крАжу в магазИне; to shut up shop закрЫть лАвочку; прекратИть дЕятельность 2. _v. 1> дЕлать покУпки (обыкн. go shopping) 2> _ам. ходИть по магазИнам, чтОбы ознакОмиться с цЕнами, присмотрЕть вещь 3> _жарг. сажАть в тюрьмУ 4> _жарг. выдавАть (сообщника); shop around _ам. искАть рабОту, мЕсто 
 shop floor  [хS█pfl█Ы] _n. профсоЮзные мАссы; рядовЫе члЕны профсоЮза; on the shop floor средИ рядовЫх члЕнов профсоЮза, на Уровне первИчной профсоЮзной организАции 
 shop hours  [хS█pau╚z] _n. _pl. врЕмя рабОты магазИнов 
 shop window  [хS█pхwInd╚u] _n. витрИна; to have everything in the shop window, to have all one's goods in the shop window а> быть повЕрхностным человЕком; б> выставлЯть всЁ напокАз 
Index: mueller/Mueller7GPL.koi
===================================================================
--- mueller.orig/Mueller7GPL.koi	2009-02-17 13:59:50.000000000 +0600
+++ mueller/Mueller7GPL.koi	2009-02-17 13:59:56.000000000 +0600
@@ -36860,7 +36860,7 @@
 shootinggallery  [хSuЫtINгgQl╚rI] _n. тир
 shootingrange  [хSuЫtINreIndZ] _n. стрельбище, полигон
 shootingstick  [хSuЫtINstIk] _n. трость-сиденье
-shop  [S█p] 1. _n. 1> лавка, магазин 2> мастерская, цех; closed shop _ам. предприятие, принимающее на работу только членов профсоюза; open shop _ам. предприятие, принимающее на работу как членов, так и нечленов профсоюза 3> _разг. заведение, учреждение, предприятие 4> профессия, занятие; дела, вопросы, темы, связанные с чьей-л. профессией; stop thinking of shop! хватит думать о делах или работе!; to talk shop говорить о делах, говорить на узкопрофессиональные темы во время общего разговора (в гостях и т.п.) 5> _attr. цеховой; shop committee цеховой комитет; shop chairman _ам. цеховой староста; all over the shop разбросанный повсюду, в беспорядке; to come to the wrong shop обратиться не по адресу; to get a shop _театр. получить ангажемент; to lift a shop совершить кражу в магазине; to shut up shop закрыть лавочку; прекратить деятельность 2. _v. 1> делать покупки (обыкн. go shop ping) 2> _ам. ходить по магазинам, чтобы ознакомиться с ценами, присмотреть вещь 3> _жарг. сажать в тюрьму 4> _жарг. выдавать (сообщника); shop around _ам. искать работу, место
+shop  [S█p] 1. _n. 1> лавка, магазин 2> мастерская, цех; closed shop _ам. предприятие, принимающее на работу только членов профсоюза; open shop _ам. предприятие, принимающее на работу как членов, так и нечленов профсоюза 3> _разг. заведение, учреждение, предприятие 4> профессия, занятие; дела, вопросы, темы, связанные с чьей-л. профессией; stop thinking of shop! хватит думать о делах или работе!; to talk shop говорить о делах, говорить на узкопрофессиональные темы во время общего разговора (в гостях и т.п.) 5> _attr. цеховой; shop committee цеховой комитет; shop chairman _ам. цеховой староста; all over the shop разбросанный повсюду, в беспорядке; to come to the wrong shop обратиться не по адресу; to get a shop _театр. получить ангажемент; to lift a shop совершить кражу в магазине; to shut up shop закрыть лавочку; прекратить деятельность 2. _v. 1> делать покупки (обыкн. go shopping) 2> _ам. ходить по магазинам, чтобы ознакомиться с ценами, присмотреть вещь 3> _жарг. сажать в тюрьму 4> _жарг. выдавать (сообщника); shop around _ам. искать работу, место
 shop floor  [хS█pfl█Ы] _n. профсоюзные массы; рядовые члены профсоюза; on the shop floor среди рядовых членов профсоюза, на уровне первичной профсоюзной организации
 shop hours  [хS█pau╚z] _n. _pl. время работы магазинов
 shop window  [хS█pхwInd╚u] _n. витрина; to have everything in the shop window, to have all one's goods in the shop window а> быть поверхностным человеком; б> выставлять всё напоказ
